lesson_15/jamirabailey/ts-node-express/src/index.ts

```diff
@@ -0,0 +1,16 @@
+// src/index.js
+import express, { Express, Request, Response } from "express";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const app: Express = express();
+const port = process.env.PORT || 3000;
+
+app.get("/", (req: Request, res: Response) => {
```

Comment:
You should use express static middleware to host the `public` folder I recommended you create as a replacement for this function call.